src/main/kotlin/inc/evil/d17map/services/UserService.kt

```diff
@@ -1,4 +1,29 @@
 package inc.evil.d17map.services
 
-class UserService {
-}
\ No newline at end of file
+import inc.evil.d17map.dtos.UserDto
+import inc.evil.d17map.entities.User
+import inc.evil.d17map.repositories.UserRepository
+import org.springframework.stereotype.Service
+
+@Service
+class UserService(
+    private val userRepository: UserRepository,
+) {
+    fun createUser(userDto: UserDto): UserDto {
```

Comment:
Ta metoda, jak i cały serwis mogą zostać usunięte, z uwagi na to, iż nadchodzi Security Update 💯 , który zarządza całym cyklem istenienia użytkownika!